tests/test_solver.py

```diff
@@ -694,3 +694,13 @@ def test_python_site_packages_path(tmp_env: TmpEnvFixture) -> None:
                 assert prec.python_site_packages_path == "lib/python3.13t/site-packages"
         else:
             assert prec.python_site_packages_path is None
+
+
+@pytest.mark.parametrize("shards", (True, False))
```

Comment:
I see the https://github.com/conda/conda-libmamba-solver/blob/main/tests/test_shards.py#L88 fixture only supports shards-enabled unfortunately